warehouse/models/mart/ntd_annual_reporting/fct_funding_sources_by_expense_type.sql

```diff
@@ -3,9 +3,25 @@ WITH staging_funding_sources_by_expense_type AS (
     FROM {{ ref('stg_ntd__funding_sources_by_expense_type') }}
 ),
 
+current_dim_organizations AS (
+    SELECT
+        ntd_id,
+        caltrans_district
+    FROM {{ ref('dim_organizations') }}
+    WHERE _is_current
+),
+
+enrich_with_caltrans_district AS (
+    SELECT
+        staging_funding_sources_by_expense_type.*,
+        current_dim_organizations.caltrans_district
+    FROM staging_funding_sources_by_expense_type
+    LEFT JOIN current_dim_organizations USING (ntd_id)
```

Comment:
I think it would be a good idea to make the `current_dim_organizations` table with the Caltrans District plus NTD an intermediate table since this is being added in a lot of places.